ext/drbbdup/drbbdup.c

```diff
@@ -404,7 +415,8 @@ drbbdup_set_up_copies(void *drcontext, instrlist_t *bb, drbbdup_manager_t *manag
      * are live at the start of bb copies, unless there is other control
      * flow from prior expansions such as drutil_expand_rep_string().
      */
-    drreg_set_bb_properties(drcontext, DRREG_IGNORE_CONTROL_FLOW);
+    if (!has_prior_control_flow)
```

Comment:
Comment that we are turning off optimisation but this is necessary due to other control flow.

Reply:
(for this bb)

Reply:
Done.